src/main/java/com/interfaceict/ptrakcer/services/utils/Utils.java

```diff
@@ -0,0 +1,19 @@
+//
+// Created by htrap19 on 1/8/22
+//
+
+package com.interfaceict.ptrakcer.services.utils;
+
+public class Utils
+{
+    static public <T> Boolean CheckIfItsNullOrEmpty(T obj)
```

Comment:
There are multiple validation rules provided by the `spring-boot-starter-validation` dependency. 
For example, it's possible to replace `CheckIfItsNullOrEmpty` validation function with just two annotation:
`@NotEmpty` or `@NotNull`